tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/utils/ProjectHasNature.java

```diff
@@ -45,6 +46,13 @@ public boolean test() {
 		pd.open();
 		new WaitUntil(new ShellIsAvailable("Properties for " + projectName), TimePeriod.DEFAULT);
 		new DefaultTreeItem("Project Facets").select();
+		if (!getJavaVersion().equals("1.8")) {
+			try {
+				new WaitUntil(new ShellIsAvailable("Setting Java Build Path"), TimePeriod.DEFAULT);
```

Comment:
You could use new DefaultShell("Setting Java Build Path");
but approaches are valid.

Reply:
Changed.